app/views/pages/index.haml

```diff
@@ -1,6 +1,5 @@
-.center.jumbotron
-  %h1
-    Welcome to #{Rails.application.config.application_name}
-  %p
-    This is the home page for the #{link_to Rails.application.config.application_name, root_path} app.
-  = link_to "Sign up now!", new_user_registration_path, class: "btn btn-lg btn-primary"
\ No newline at end of file
+%section#welcome.container
+  %h1=t("welcome_h1")
+  %p=t("welcome_intro")
+  -#  This is the home page for the #{link_to Rails.application.config.application_name, root_path} app.
```

Comment:
Files should end with a trailing newline<br>Line is too long. [89/80]<br>Comment should have a space after the `#`